packages/app-bridge/src/registries/api/GetAssetBulkDownloadToken.ts

```diff
@@ -1,8 +1,10 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import { Asset } from '../../types';
+
 export type GetAssetBulkDownloadTokenPayload = {
     documentBlockId: number;
```

Comment:
Same comment as [here](https://github.com/Frontify/web-app/pull/2037#discussion_r1409019508), users shouldn't have to know the block ID.